openpype/pype_commands.py

```diff
@@ -403,7 +403,10 @@ def get_value(get_url, project=None):
 
         if general_location == "keyring":
             registry = OpenPypeSecureRegistry(path_part)
-            return registry.get_item(key)
+            try:
```

Comment:
Should we skip the exception and pretend that nothing happened? Result will be None which would look like the value is not set. But we don't know if is set because it can't be accessed.

Reply:
`get_secure_value` in lib throws an exception if key not found.
Cli catches the exception and prints it on stdout.